app/src/main/res/values/strings.xml

```diff
@@ -321,6 +321,7 @@
   <string name="admin_settings_enter_user_new_pin">%1$s\'s New PIN.</string>
   <!-- ResetPinDialogFragment -->
   <string name="reset_pin_enter">Enter a New Pin</string>
+  <string name="reset_pin_enter_dialog_message">This PIN will be %s\'s new PIN and will be required when signing in.</string>
```

Comment:
@BenHenning Similar strings will be required in other `strings.xml` file too, but do PR author takes care of that or its a separate process?

Reply:
It's a separate process (our Translatewiki volunteers will translate this string once it's checked in).